src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const max = 10;
+const min = 2;
+
+const setButton = (el, state) => (el.disabled = state);
+
+appendRow.addEventListener('click', (e) => {
+  const rows = [...tbody.querySelectorAll('tr')];
+
+  rows.push(rows[1].cloneNode(true));
```

Comment:
It's not a good practice to directly manipulate the array you are iterating over (in this case, the `rows` array). This can lead to unexpected behavior. Instead, you should create a new array or a clone of the original one and work with that.